content/en/about/case-studies/_index.md

```diff
@@ -7,4 +7,6 @@ aliases:
 doc_type: about
 type: case-studies
 sidebar_none: true
----
\ No newline at end of file
+---
+
+[comment]: <> (To add yourself as an Istio user, please edit /data/companies.yaml and add your logo to /static/logos.)
```

Comment:
Do we need to also mention updating the appropriate section of the `companies.yaml`?

Reply:
Yeah, I'm actually leaning towards having a full instructions page to point people to.  Hold this thought.